### ISSUE TITLE
Missing fields in response to `ListObjectsV2`

### ISSUE THREAD

**Issue by Raz-Hemo** [paolobarbolini/rusty-s3#49]
Using Google cloud storage, the response to `ListObjectsV2` sometimes does not contain the `StorageClass` field. This causes a failure to parse the XML response into the `ListObjectsV2Response` struct.

```
Error: ListObjectsV2: Failed parsing response

Caused by:
    missing field `StorageClass`
```

This can easily be solved by marking said field of `ListObjectsContent` as `Option`:
```
#[derive(Debug, Clone, Deserialize)]
pub struct ListObjectsContent {
    // ...
    // ...
    pub storage_class: Option<String>,
}
```

**Comment by paolobarbolini** [comment]
:thinking: I think I'm starting to understand why some S3 clients have all fields as nullable. How do you make an API that sometimes doesn't known what some of the object metadata is is beyond me.

I'll look into this in the following days.

**Comment by paolobarbolini** [comment]
Fixed by #50
